### ISSUE TITLE
Installation issue: doxygen

### ISSUE THREAD

**Issue by omsai** [LLNL/spack#45082]
### Steps to reproduce the issue

@sethrj The underlying issue is that the `patch()` function does not run at all when it is supposed to fix `./cmake/FindIconv.cmake` because of the incorrect conditional that returns early. It compiles fine after removing that return early conditional:

```diff
diff --git a/var/spack/repos/builtin/packages/doxygen/package.py b/var/spack/repos/builtin/packages/doxygen/package.py
index 75823e8846..32cd5fee9e 100644
--- a/var/spack/repos/builtin/packages/doxygen/package.py
+++ b/var/spack/repos/builtin/packages/doxygen/package.py
@@ -130,8 +130,6 @@ def determine_variants(cls, exes, version_str):
     conflicts("%gcc@7.0:7.9", when="@1.9:")
 
     def patch(self):
-        if self.spec["iconv"].name != "libiconv":
-            return
         # On Linux systems, iconv is provided by libc. Since CMake finds the
         # symbol in libc, it does not look for libiconv, which leads to linker
         # errors. This makes sure that CMake always looks for the external
``` 

```console
$ spack spec doxygen
Input spec
--------------------------------
 -   doxygen

Concretized
--------------------------------
[+]  doxygen@1.11.0%gcc@10.2.1~graphviz~ipo~mscgen build_system=cmake build_type=Release generator=make patches=8b46b76 arch=linux-pureos10-skylake
[+]      ^bison@3.8.2%gcc@10.2.1~color build_system=autotools arch=linux-pureos10-skylake
[+]          ^m4@1.4.19%gcc@10.2.1+sigsegv build_system=autotools patches=9dc5fbd,bfdffa7 arch=linux-pureos10-skylake
[+]              ^libsigsegv@2.14%gcc@10.2.1 build_system=autotools arch=linux-pureos10-skylake
[+]      ^cmake@3.27.9%gcc@10.2.1~doc+ncurses+ownlibs build_system=generic build_type=Release arch=linux-pureos10-skylake
[+]          ^curl@8.7.1%gcc@10.2.1~gssapi~ldap~libidn2~librtmp~libssh~libssh2+nghttp2 build_system=autotools libs=shared,static tls=openssl arch=linux-pureos10-skylake
[+]              ^nghttp2@1.61.0%gcc@10.2.1 build_system=autotools arch=linux-pureos10-skylake
[+]              ^perl@5.38.2%gcc@10.2.1+cpanm+opcode+open+shared+threads build_system=generic patches=714e4d1 arch=linux-pureos10-skylake
[+]                  ^berkeley-db@18.1.40%gcc@10.2.1+cxx~docs+stl build_system=autotools patches=26090f4,b231fcc arch=linux-pureos10-skylake
[+]          ^ncurses@6.5%gcc@10.2.1~symlinks+termlib abi=none build_system=autotools patches=7a351bc arch=linux-pureos10-skylake
[+]          ^zlib-ng@2.1.6%gcc@10.2.1+compat+new_strategies+opt+pic+shared build_system=autotools arch=linux-pureos10-skylake
[+]      ^flex@2.6.3%gcc@10.2.1+lex~nls build_system=autotools arch=linux-pureos10-skylake
[+]          ^diffutils@3.10%gcc@10.2.1 build_system=autotools arch=linux-pureos10-skylake
[+]          ^findutils@4.9.0%gcc@10.2.1 build_system=autotools patches=440b954 arch=linux-pureos10-skylake
[+]      ^gcc-runtime@10.2.1%gcc@10.2.1 build_system=generic arch=linux-pureos10-skylake
[e]      ^glibc@2.31%gcc@10.2.1 build_system=autotools arch=linux-pureos10-skylake
[+]      ^gmake@4.4.1%gcc@10.2.1~guile build_system=generic arch=linux-pureos10-skylake
[+]      ^libiconv@1.17%gcc@10.2.1 build_system=autotools libs=shared,static arch=linux-pureos10-skylake
[+]      ^python@3.11.9%gcc@10.2.1+bz2+crypt+ctypes+dbm~debug+libxml2+lzma~nis~optimizations+pic+pyexpat+pythoncmd+readline+shared+sqlite3+ssl~tkinter+uuid+zlib build_system=generic patches=13fa8bf,b0615b2,ebdca64,f2fd060 arch=linux-pureos10-skylake
[+]          ^bzip2@1.0.8%gcc@10.2.1~debug~pic+shared build_system=generic arch=linux-pureos10-skylake
[+]          ^expat@2.6.2%gcc@10.2.1+libbsd build_system=autotools arch=linux-pureos10-skylake
[+]              ^libbsd@0.12.2%gcc@10.2.1 build_system=autotools arch=linux-pureos10-skylake
[+]                  ^libmd@1.0.4%gcc@10.2.1 build_system=autotools arch=linux-pureos10-skylake
[+]          ^gdbm@1.23%gcc@10.2.1 build_system=autotools arch=linux-pureos10-skylake
[+]          ^gettext@0.22.5%gcc@10.2.1+bzip2+curses+git~libunistring+libxml2+pic+shared+tar+xz build_system=autotools arch=linux-pureos10-skylake
[+]              ^libxml2@2.10.3%gcc@10.2.1+pic~python+shared build_system=autotools arch=linux-pureos10-skylake
[+]              ^tar@1.34%gcc@10.2.1 build_system=autotools zip=pigz arch=linux-pureos10-skylake
[+]                  ^pigz@2.8%gcc@10.2.1 build_system=makefile arch=linux-pureos10-skylake
[+]                  ^zstd@1.5.6%gcc@10.2.1+programs build_system=makefile compression=none libs=shared,static arch=linux-pureos10-skylake
[+]          ^libffi@3.4.6%gcc@10.2.1 build_system=autotools arch=linux-pureos10-skylake
[+]          ^libxcrypt@4.4.35%gcc@10.2.1~obsolete_api build_system=autotools patches=4885da3 arch=linux-pureos10-skylake
[+]          ^openssl@3.3.0%gcc@10.2.1~docs+shared build_system=generic certs=mozilla arch=linux-pureos10-skylake
[+]              ^ca-certificates-mozilla@2023-05-30%gcc@10.2.1 build_system=generic arch=linux-pureos10-skylake
[+]          ^pkgconf@2.2.0%gcc@10.2.1 build_system=autotools arch=linux-pureos10-skylake
[+]          ^readline@8.2%gcc@10.2.1 build_system=autotools patches=bbf97f1 arch=linux-pureos10-skylake
[+]          ^util-linux-uuid@2.40.1%gcc@10.2.1 build_system=autotools arch=linux-pureos10-skylake
[+]          ^xz@5.4.6%gcc@10.2.1~pic build_system=autotools libs=shared,static arch=linux-pureos10-skylake
[+]      ^spdlog@1.14.1%gcc@10.2.1~ipo+shared build_system=cmake build_type=Release generator=make arch=linux-pureos10-skylake
[+]          ^fmt@11.0.0%gcc@10.2.1~ipo+pic~shared build_system=cmake build_type=Release cxxstd=11 generator=make arch=linux-pureos10-skylake
[+]      ^sqlite@3.43.2%gcc@10.2.1+column_metadata+dynamic_extensions+fts~functions+rtree build_system=autotools arch=linux-pureos10-skylake
```

### Error message

<details><summary>Error message</summary>

<pre>
==> Installing doxygen-1.11.0-cpaksqqh3ubrjb6bdxt2vuebgmitvtfl [44/44]
==> No binary for doxygen-1.11.0-cpaksqqh3ubrjb6bdxt2vuebgmitvtfl found: installing from source
==> Using cached archive: /home/omsai/src/spack/var/spack/cache/_source-cache/archive/1f/1fea49c69e51fec3dd2599947f6d48d9b1268bd5115b1bb08dffefc1fd5d19ee.tar.gz
==> Using cached archive: /home/omsai/src/spack/var/spack/cache/_source-cache/archive/8b/8b46b763b3f0a2726f765141cbfa3eb6efd746531a4d689531e42ff56fc334e2
==> Applied patch https://github.com/doxygen/doxygen/commit/83de58c5f4f685a129127c2501f4fccd9557f6c4.patch?full_index=1
==> Ran patch() for doxygen
==> doxygen: Executing phase: 'cmake'
==> Error: ProcessError: Command exited with status 1:
    '/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/cmake-3.27.9-4byvxnde7fiircr6euirut2ecutbyo5w/bin/cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/doxygen-1.11.0-cpaksqqh3ubrjb6bdxt2vuebgmitvtfl' '-DCMAKE_BUILD_TYPE:STRING=Release' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_POLICY_DEFAULT_CMP0090:STRING=NEW' '-DCMAKE_FIND_USE_PACKAGE_REGISTRY:BOOL=OFF' '-DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON' '-DPYTHON_EXECUTABLE:STRING=/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/python-3.11.9-2sth24tlzgpsiimahe5kwlqzhnwfw5rk/bin/python3.11' '-DPython_EXECUTABLE:STRING=/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/python-3.11.9-2sth24tlzgpsiimahe5kwlqzhnwfw5rk/bin/python3.11' '-DPython3_EXECUTABLE:STRING=/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/python-3.11.9-2sth24tlzgpsiimahe5kwlqzhnwfw5rk/bin/python3.11' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=ON' '-DCMAKE_INSTALL_RPATH:STRING=/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/doxygen-1.11.0-cpaksqqh3ubrjb6bdxt2vuebgmitvtfl/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/doxygen-1.11.0-cpaksqqh3ubrjb6bdxt2vuebgmitvtfl/lib64;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/gcc-runtime-10.2.1-fwh7vunk4syrzokqgyihrhpudzbqn7bs/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/libiconv-1.17-jqy7kcpukl7agry3f5roku4xgur3y5rd/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/spdlog-1.14.1-26dnx4qmauqitb5ehjvcohawsdjdibyi/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/fmt-11.0.0-g2u5fcn7nkddhtmiykhm6foevau4rzzc/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/sqlite-3.43.2-ns423hnt3ou4efv2ztfjde65hznlhwun/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/readline-8.2-ldvccmjt4v43a3dblec3few25hqnxaqf/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/ncurses-6.5-ivon7g5s7dnxflkfuo2nnpdh6s7ixrho/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/zlib-ng-2.1.6-f54725fau7zd7owawktq7w2lxfkb7h56/lib' '-DCMAKE_PREFIX_PATH:STRING=/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/spdlog-1.14.1-26dnx4qmauqitb5ehjvcohawsdjdibyi;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/fmt-11.0.0-g2u5fcn7nkddhtmiykhm6foevau4rzzc;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/python-3.11.9-2sth24tlzgpsiimahe5kwlqzhnwfw5rk;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/sqlite-3.43.2-ns423hnt3ou4efv2ztfjde65hznlhwun;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/libiconv-1.17-jqy7kcpukl7agry3f5roku4xgur3y5rd;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/flex-2.6.3-s2rvxr6u4naqzvn2hx62pgaldr53mc55;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/cmake-3.27.9-4byvxnde7fiircr6euirut2ecutbyo5w;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/zlib-ng-2.1.6-f54725fau7zd7owawktq7w2lxfkb7h56;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/readline-8.2-ldvccmjt4v43a3dblec3few25hqnxaqf;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/ncurses-6.5-ivon7g5s7dnxflkfuo2nnpdh6s7ixrho;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/bison-3.8.2-25pt6ypijq6viohj7uss2ahvnhvgkob5;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/gmake-4.4.1-knxdqnkcpfxuwdv5j5vs5trlmowph6bi;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/gcc-runtime-10.2.1-fwh7vunk4syrzokqgyihrhpudzbqn7bs' '-Duse_sys_spdlog:BOOL=ON' '-Duse_sys_sqlite3:BOOL=ON' '/tmp/omsai/spack-stage/spack-stage-doxygen-1.11.0-cpaksqqh3ubrjb6bdxt2vuebgmitvtfl/spack-src'

1 error found in build log:
     25    -- Found FLEX: /home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/flex-2.6.3-s2rvxr6u4naqzvn2hx62pgaldr53mc55/bin/flex (
           found version "2.6.3")
     26    -- Found BISON: /home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/bison-3.8.2-25pt6ypijq6viohj7uss2ahvnhvgkob5/bin/biso
           n (found version "3.8.2")
     27    -- Looking for iconv_open
     28    -- Looking for iconv_open - found
     29    -- Performing Test ICONV_COMPILES
     30    -- Performing Test ICONV_COMPILES - Failed
  >> 31    CMake Error at /home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/cmake-3.27.9-4byvxnde7fiircr6euirut2ecutbyo5w/share/cm
           ake-3.27/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
     32      Could NOT find Iconv (missing: ICONV_COMPILES)
     33    Call Stack (most recent call first):
     34      /home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/cmake-3.27.9-4byvxnde7fiircr6euirut2ecutbyo5w/share/cmake-3.27/Modu
           les/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
     35      cmake/FindIconv.cmake:87 (find_package_handle_standard_args)
     36      CMakeLists.txt:194 (find_package)
     37    

See build log for details:
  /tmp/omsai/spack-stage/spack-stage-doxygen-1.11.0-cpaksqqh3ubrjb6bdxt2vuebgmitvtfl/spack-build-out.txt

==> Error: doxygen-1.11.0-cpaksqqh3ubrjb6bdxt2vuebgmitvtfl: ProcessError: Command exited with status 1:
    '/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/cmake-3.27.9-4byvxnde7fiircr6euirut2ecutbyo5w/bin/cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/doxygen-1.11.0-cpaksqqh3ubrjb6bdxt2vuebgmitvtfl' '-DCMAKE_BUILD_TYPE:STRING=Release' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_POLICY_DEFAULT_CMP0090:STRING=NEW' '-DCMAKE_FIND_USE_PACKAGE_REGISTRY:BOOL=OFF' '-DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON' '-DPYTHON_EXECUTABLE:STRING=/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/python-3.11.9-2sth24tlzgpsiimahe5kwlqzhnwfw5rk/bin/python3.11' '-DPython_EXECUTABLE:STRING=/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/python-3.11.9-2sth24tlzgpsiimahe5kwlqzhnwfw5rk/bin/python3.11' '-DPython3_EXECUTABLE:STRING=/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/python-3.11.9-2sth24tlzgpsiimahe5kwlqzhnwfw5rk/bin/python3.11' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=ON' '-DCMAKE_INSTALL_RPATH:STRING=/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/doxygen-1.11.0-cpaksqqh3ubrjb6bdxt2vuebgmitvtfl/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/doxygen-1.11.0-cpaksqqh3ubrjb6bdxt2vuebgmitvtfl/lib64;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/gcc-runtime-10.2.1-fwh7vunk4syrzokqgyihrhpudzbqn7bs/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/libiconv-1.17-jqy7kcpukl7agry3f5roku4xgur3y5rd/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/spdlog-1.14.1-26dnx4qmauqitb5ehjvcohawsdjdibyi/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/fmt-11.0.0-g2u5fcn7nkddhtmiykhm6foevau4rzzc/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/sqlite-3.43.2-ns423hnt3ou4efv2ztfjde65hznlhwun/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/readline-8.2-ldvccmjt4v43a3dblec3few25hqnxaqf/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/ncurses-6.5-ivon7g5s7dnxflkfuo2nnpdh6s7ixrho/lib;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/zlib-ng-2.1.6-f54725fau7zd7owawktq7w2lxfkb7h56/lib' '-DCMAKE_PREFIX_PATH:STRING=/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/spdlog-1.14.1-26dnx4qmauqitb5ehjvcohawsdjdibyi;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/fmt-11.0.0-g2u5fcn7nkddhtmiykhm6foevau4rzzc;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/python-3.11.9-2sth24tlzgpsiimahe5kwlqzhnwfw5rk;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/sqlite-3.43.2-ns423hnt3ou4efv2ztfjde65hznlhwun;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/libiconv-1.17-jqy7kcpukl7agry3f5roku4xgur3y5rd;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/flex-2.6.3-s2rvxr6u4naqzvn2hx62pgaldr53mc55;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/cmake-3.27.9-4byvxnde7fiircr6euirut2ecutbyo5w;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/zlib-ng-2.1.6-f54725fau7zd7owawktq7w2lxfkb7h56;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/readline-8.2-ldvccmjt4v43a3dblec3few25hqnxaqf;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/ncurses-6.5-ivon7g5s7dnxflkfuo2nnpdh6s7ixrho;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/bison-3.8.2-25pt6ypijq6viohj7uss2ahvnhvgkob5;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/gmake-4.4.1-knxdqnkcpfxuwdv5j5vs5trlmowph6bi;/home/omsai/src/spack/opt/spack/linux-pureos10-skylake/gcc-10.2.1/gcc-runtime-10.2.1-fwh7vunk4syrzokqgyihrhpudzbqn7bs' '-Duse_sys_spdlog:BOOL=ON' '-Duse_sys_sqlite3:BOOL=ON' '/tmp/omsai/spack-stage/spack-stage-doxygen-1.11.0-cpaksqqh3ubrjb6bdxt2vuebgmitvtfl/spack-src'
==> Error: doxygen-1.11.0-cpaksqqh3ubrjb6bdxt2vuebgmitvtfl: Package was not installed
==> Error: Installation request failed.  Refer to reported errors for failing package(s).
</pre></details>

### Information on your system

* **Spack:** 0.23.0.dev0 (aeaa922eef45b14199b486fa2c188252df8c9544)
* **Python:** 3.9.2
* **Platform:** linux-pureos10-skylake
* **Concretizer:** clingo

### Additional information

- [spack-build-out.txt](https://github.com/user-attachments/files/16117128/spack-build-out.txt)
- [spack-build-env.txt](https://github.com/user-attachments/files/16117129/spack-build-env.txt)
- [CMakeConfigureLog.yaml](https://github.com/user-attachments/files/16117135/CMakeConfigureLog.yaml.txt)

### General information

- [X] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [X] I have run `spack maintainers <name-of-the-package>` and **@mentioned** any maintainers
- [X] I have uploaded the build log and environment files
- [X] I have searched the issues of this repo and believe this is not a duplicate

**Comment by sethrj** [comment]
@haampie this might be related to #43996  ... thoughts?

**Comment by haampie** [comment]
It looks fine. Please share more details

**Comment by haampie** [comment]
Actually the bug was introduced in https://github.com/spack/spack/pull/35322.

Fix in #45090

**Comment by alalazo** [comment]
Fixed
